### ISSUE TITLE
Git Bash 2.20.0-rc0 opens two consoles

### ISSUE THREAD

**Issue by mloskot** [git-for-windows/git#1942]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

- Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options
git version 2.20.0.rc0.windows.1
cpu: x86_64
built from commit: 41dcf8176e60f5b816f293845e3424a7f4509f0b
sizeof-long: 4
sizeof-size_t: 8
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver
Microsoft Windows [Version 10.0.17134.376]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
Editor Option: VIM
Custom Editor Path:
Path Option: BashOnly
SSH Option: OpenSSH
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Enabled
Enable Builtin Rebase: Disabled
Enable Builtin Stash: Disabled
```

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

START > Git > Git Bash

 - What actually happened?

First, a blank command prompt opens, then Git Bash console opens. The two consoles remain opened. See the issue in action in the screencast below (links to YouTube video):

[![Git for Windows 2.20.0.rc0: Git Bash opens two consoles](http://img.youtube.com/vi/-809r4Iv8z0/0.jpg)](https://www.youtube.com/watch?v=-809r4Iv8z0 "Git for Windows 2.20.0.rc0: Git Bash opens two consoles")

**Comment by dscho** [comment]
Fixed via 5f26ad6184e30d622d14aefe18016a97e089ce85 (see also the just-released Git for Windows v2.19.2).

**Comment by mloskot** [comment]
Thank you!
